nerfstudio/data/pixel_samplers.py

```diff
@@ -111,7 +111,7 @@ def collate_image_dataset_batch_equirectangular(batch: Dict, num_rays_per_batch:
     num_images, image_height, image_width, _ = batch["image"].shape
 
     # only sample within the mask, if the mask is in the batch
-    if "mask" in batch:
+    if "mask" in batch:  # pylint: disable=no-else-raise
```

Comment:
Remove else to fix pylint error
https://pylint.pycqa.org/en/latest/user_guide/messages/refactor/no-else-raise.html